powershell-adapter/powershell.resource.ps1

```diff
@@ -1,427 +1,349 @@
-# Copyright (c) Microsoft Corporation.
```

Comment:
We need to keep the copyright notice on all our source files

Reply:
fixed

Reply:
The diff still shows the copyright header as deleted